litt/table.go

```diff
@@ -0,0 +1,96 @@
+package litt
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigenda/litt/types"
+)
+
+// Table is a key-value store with a namespace that does not overlap with other tables.
+// Values may be written to the table, but once written, they may not be changed or deleted (except via TTL).
+type Table interface {
```

Comment:
I assume all methods here are thread-safe for Table?

Reply:
Yes. Added

```
// All methods in this interface are thread safe.
```